l10n_fr_siret_account/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    siren = fields.Char(tracking=50)
+    nic = fields.Char(tracking=51)
```

Comment:
I don't get why you redefine this two fields on the glue module. Why not in base module ?

Reply:
tracking is a feature of the mail module, not base. l10n_fr_siret_account depend on account which depend on mail.
For example, you will not see tracking=X on res.partner fields defined in the base module... they are in the mail module: https://github.com/odoo/odoo/blob/18.0/addons/mail/models/res_partner.py#L19